### DISCUSSION TITLE
Log severity of  BLE OnCharacteristic event

### DISCUSSION THREAD

**Discussion by Corsatobi** [shinyorg/shiny#1432] [Q&A]
Hi,
is there any specific reason why BLE characteristic events like OnCharacteristicRead,OnCharacteristicwrite and OnCharacteristicChanged have the log severity "Warning"?
I am just asking for interest, as i am using serilog in my app with a severity of "Warning" to write only in the log if something more or less bad or irregular happens, a characteristicEvent is surely nothing of this. 
or is there a way to change shiny.BLEs logger severity somehow?

Thank you very much in advance.

**Comment by aritchie** [comment]
Likely just a typo.  Just send a PR in the future
